projects/angular4-paystack/src/lib/angular4-paystack-embed.component.ts

```diff
@@ -65,7 +65,6 @@ export class Angular4PaystackEmbed implements OnInit {
       plan: this.plan || '',
       quantity: this.quantity || '',
       subaccount: this.subaccount || '',
-      channels: this.channels || ['card', 'bank'],
```

Comment:
By removing it, users cannot pass in their required channel. This would be better

`channels: this.channels`